Library/ViewModels/RootViewModel.swift

```diff
@@ -357,14 +357,20 @@ public final class RootViewModel: RootViewModelType, RootViewModelInputs, RootVi
       }
 
     self.tabBarItemsData
+      .combineLatest(with: prevSelectedTabBarItem.signal)
```

Comment:
Unrelated to this PR, but the name of this view model doesn't indicate any kind of analytics framework `RootViewModel`. Only reason is because I just created a `RootCommentCellViewModel` and it might get a bit confusing as these two are completely unrelated.

Reply:
The analytics client is part of our `AppEnvironment` so you'll see it referenced in a bunch of different view models like this:

`AppEnvironment.current.ksrAnalytics`